activities.json

```diff
@@ -317,7 +317,7 @@
   },
   {
     "ciuName": null,
-    "description": "This proposal adds an API for prompting and querying the user\u2019s contacts for one or more items with a handful of contact properties.",
+    "description": "This proposal adds an API for prompting and querying the user’s contacts for one or more items with a handful of contact properties.",
```

Comment:
IIRC this unfortunately breaks the activities.py tool (as per https://github.com/mozilla/standards-positions/pull/652)